### ISSUE TITLE
Improve mount manager.

### ISSUE THREAD

**Issue by edmundlaugasson** [MeanEYE/Sunflower#38]
I would expect from twin-panel filemanager also support to file transfer protocols like FTP, SFTP, FTPs, NFS, SMB, Samba, CIFS, WebDAV, AFP (Apple), NCP (Netware) etc

It means, that filemanager could have also mount manager, where I can also save these network mountpoints for everydays comfortable use. Mount manager would manage also local mounts. Such mount manager could be used also as mount bookmark like in FileZilla Client.

It also means, that user can manage also saved passwords - whether to save or not them or delete one or all of them.

Also file editing over network would be useful

Basically it means, that I could do same things like with FileZilla Client.

Currently if I use system bookmarks, where also network folders are, then Sunflower gives an error when I try to open it.
